src/modules/GameConstants.ts

```diff
@@ -2018,7 +2018,6 @@ export const TemporaryBattles = [
     'Gym Leader Bede',
     'Rampaging Gigalith',
     'Rampaging Froslass',
-    'Gym Leader Marnie',
```

Comment:
This is a constant for a temp battle, not a credit! 😰 Removing this would cause glitches in the game! 😟